### PR TITLE
Fixed scanning / issue #6

### PR DESCRIPTION
**Pull request by matanshukry**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -14,6 +14,7 @@ require (
 	github.com/graphql-go/graphql v0.7.8 // indirect
 	github.com/hashicorp/golang-lru v0.5.1 // indirect
 	github.com/ianlancetaylor/demangle v0.0.0-20181102032728-5e5cf60278f6 // indirect
+	github.com/inconshreveable/mousetrap v1.0.0 // indirect
```
Comment (bennyz) [inline go.mod]: is this needed?

File: go.sum [hunk 2]
```diff
@@ -69,6 +69,8 @@ github.com/hashicorp/hcl v1.0.0/go.mod h1:E5yfLk+7swimpb2L/Alb/PJmXilQ/rhwaUYs4T
 github.com/hpcloud/tail v1.0.0/go.mod h1:ab1qPbhIpdTxEkNHXyeSf5vhxWSCs/tWer42PpOxQnU=
 github.com/ianlancetaylor/demangle v0.0.0-20181102032728-5e5cf60278f6 h1:UDMh68UUwekSh5iP2OMhRRZJiiBccgV7axzUG8vi56c=
 github.com/ianlancetaylor/demangle v0.0.0-20181102032728-5e5cf60278f6/go.mod h1:aSSvb/t6k1mPoxDqO4vJh6VOCGPwU4O0C2/Eqndh1Sc=
+github.com/inconshreveable/mousetrap v1.0.0 h1:Z8tu5sraLXCXIcARxBp/8cbvlwVa7Z1NHg9XEKhtSvM=
```
Comment (bennyz) [inline go.sum]: same

File: go.mod (hunk 1 above)
Comment (matanshukry) [reply go.mod]: No idea, I didn't add it manually. I can guess these files are dependencies, but honestly I didn't even read about them.

File: go.sum (hunk 2 above)
Comment (matanshukry) [reply go.sum]: same

**Comment by bennyz** [comment]
Thanks for the contribution!

**Pull request merged**